terracumber_config/tf_files/MLM-5.1-SLE-update-NUE.tf

```diff
@@ -0,0 +1,232 @@
+// Mandatory variables for terracumber
+variable "URL_PREFIX" {
+  type = string
+  default = "https://ci.suse.de/view/Manager/view/Manager-5.1/job/manager-5.1-qe-sle-update-NUE"
+}
+
+// Not really used as this is for --runall parameter, and we run cucumber step by step
+variable "CUCUMBER_COMMAND" {
+  type = string
+  default = "export PRODUCT='SUSE-Manager' && run-testsuite"
+}
+
+variable "CUCUMBER_GITREPO" {
+  type = string
+  default = "https://github.com/SUSE/spacewalk.git"
+}
+
+variable "CUCUMBER_BRANCH" {
+  type = string
+  default = "Manager-5.1"
+}
+
+variable "CUCUMBER_RESULTS" {
+  type = string
+  default = "/root/spacewalk/testsuite"
+}
+
+variable "MAIL_SUBJECT" {
+  type = string
+  default = "Results 5.1 SLE Update $status: $tests scenarios ($failures failed, $errors errors, $skipped skipped, $passed passed)"
+}
+
+variable "MAIL_TEMPLATE" {
+  type = string
+  default = "../mail_templates/mail-template-jenkins.txt"
+}
+
+variable "MAIL_SUBJECT_ENV_FAIL" {
+  type = string
+  default = "Results 5.1 SLE Update: Environment setup failed"
+}
+
+variable "MAIL_TEMPLATE_ENV_FAIL" {
+  type = string
+  default = "../mail_templates/mail-template-jenkins-env-fail.txt"
+}
+
+variable "MAIL_FROM" {
+  type = string
+  default = "galaxy-noise@suse.de"
+}
+
+variable "MAIL_TO" {
+  type = string
+  default = "galaxy-noise@suse.de"
+}
+
+// sumaform specific variables
+variable "SCC_USER" {
+  type = string
+}
+
+variable "SCC_PASSWORD" {
+  type = string
+}
+
+variable "SERVER_CONTAINER_REPOSITORY" {
+  type = string
+}
+
+variable "PROXY_CONTAINER_REPOSITORY" {
+  type = string
+}
+
+variable "SERVER_CONTAINER_IMAGE" {
+  type = string
+  default = ""
+}
+
+variable "GIT_USER" {
+  type = string
+  default = null
+}
+
+variable "GIT_PASSWORD" {
+  type = string
+  default = null
+}
+
+terraform {
+  required_version = "1.0.10"
+  required_providers {
+    libvirt = {
+      source = "dmacvicar/libvirt"
+      version = "0.8.1"
+    }
+  }
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://suma-11.mgr.suse.de/system"
+}
+
+module "base" {
+  source            = "./modules/base"
+
+  cc_username       = var.SCC_USER
+  cc_password       = var.SCC_PASSWORD
+  product_version   = "5.1-released"
+  name_prefix       = "mlm-su-51-"
+  use_avahi         = false
+  domain            = "mgr.suse.de"
+  images            = [ "sles15sp6o", "opensuse155o", "opensuse156o", "slemicro55o" ]
+
+  mirror            = "minima-mirror-ci-bv.mgr.suse.de"
+  use_mirror_images = true
+
+  testsuite         = true
+
+  provider_settings = {
+    pool        = "ssd"
+    bridge      = "br1"
+  }
+}
+
+module "server_containerized" {
+  source             = "./modules/server_containerized"
+  base_configuration = module.base.configuration
+  name               = "server"
+  image              = "slemicro55o"
+  provider_settings = {
+    mac                = "aa:b2:93:01:02:79"
+    data_pool          = "ssd"
+  }
```

Comment:
I wonder what default values will be used for the server? We need at least 16 GB and 4 vCPUs.

Reply:
it looks like it's the case already ; this is a copy-paste of 5.0, and:
```
suma-su-50-server:~ # free -h
               total        used        free      shared  buff/cache   available
Mem:            15Gi       3.0Gi        11Gi       187Mi       1.8Gi        12Gi
Swap:             0B          0B          0B
```

```
suma-su-50-server:~ # lscpu
Architecture:             x86_64
  CPU op-mode(s):         32-bit, 64-bit
  Address sizes:          42 bits physical, 48 bits virtual
  Byte Order:             Little Endian
CPU(s):                   4
```